foodnet/urls_allauth.py

```diff
@@ -0,0 +1,34 @@
+"""
+allauth urlconf where some of the views are replaced by foodnet.membership
+views.
+
+HAS TO BE INCLUDED SEPARATELY BECAUSE ALLAUTH DOESN'T SUPPORT BEING
+IN OUR FOODNET URLCONF
```

Comment:
No need for yelling :smiley: I would say this comment is unnecessary. Let's explain it in the setup part of the documentation. (Unless you really feel strongly about it being there.)